qa.gemspec

```diff
@@ -22,6 +22,7 @@ Gem::Specification.new do |s|
   s.add_dependency "nokogiri", "~> 1.6.0"
   s.add_dependency "activerecord-import"
   s.add_dependency "deprecation"
+  s.add_dependency "linkeddata"
```

Comment:
This seems like a big dependency to pull in, and it may not ever be used. Can we just require the application provide whatever format readers they need?